doc/source/install/docker.rst

```diff
@@ -1,28 +1,85 @@
-######################
-Docker virtual machine
-######################
+#################
+Docker containers 
+#################
 
 Qserv is also available as a Docker image (see https://www.docker.com/).
 
-***************
-Basic use cases
-***************
+.. note::
+
+   This procedure was tested with Docker 1.6 and 1.8.
+
+***********************
+Build main Qserv images
+***********************
 
-Create a docker image containing a Qserv mono-node instance and run integration tests:
+Create two images:
+
+- qserv:latest: Qserv latest release
+- qserv:dev: Allow to build/configure and run Qserv from sources
 
 .. code-block:: bash
 
    . /path/to/lsst/stack/loadLSST.bash
    cd ${SRC_DIR}/qserv/admin/tools/docker
-   ./build-docker-img.sh centos_7
+   ./build-images.sh
+   
+*********
+Use cases
+*********
 
-Dockerfile example for centos7:
-.. literalinclude:: ../../../admin/tools/docker/centos_7/Dockerfile
-   :language: bash
-   :linenos:
+Test latest release
+===================
+
+Run mono-node integration test against latest Qserv release:
+
+.. code-block:: bash
+
+   docker run -it --rm \
+   -h $(hostname)-docker -u qserv \
+   fjammes/qserv:latest \
+   /qserv/scripts/mono-node-test.sh
+
+Develop and test 
+================
+
+Set uid in qserv:dev w.r.t your host machine user account. This creates a qserv:dev-uid image which can mount host source code in a container with correct permissions. 
+Host might be a development machine or a continuous integration server.
+
+.. code-block:: bash
+
+   ./change-uid.sh
+ 
```

Comment:
Hmm, okay for now I guess...   Could one just -v in /etc/group, /etc/passwd, and use --user as an alternative for dev instead of spinning a whole add'l container?  User namespacing is supposed to be coming to docker "real soon now" (1.9?)